draft-ietf-quic-multipath.md

```diff
@@ -592,25 +592,23 @@ as "no packet received on any path for the duration of the idle timeout".
 When only one path is available, servers MUST follow the specifications
 in {{QUIC-TRANSPORT}}.
 
-When more than one path is available, hosts shall monitor the arrival of
-packets and acknowledgements for packets sent over each
-path. Hosts MAY consider closing a path if for at least the period of the
-idle timeout as specified in {{Section 10.1. of QUIC-TRANSPORT}}
-(a) no packet was received or (b) no packet sent over this path was acknowledged.
-Endpoints that desire to close a path because of the idle timer rule
-MUST do so explicitly by sending a PATH_ABANDON frame on another path,
-as defined in {{path-close}}.
-
-To avoid idle timeout of a path, endpoints
+This dcoument does not specify any specific per-path timeouts. An endpoint
+can decide to close a path at any time, whether the path is in active
+use or not, by sending a PATH_ABANDON frame. Particularly, it is not required
+to send a PATH_ABANDON frame at any specific point in time if that would be
+the only frame to send.
```

Comment:
Typo dcoument 

I think we could rewrite the second sentence a little bit. Maybe: 
"It is not required to send a PATH_ABANDON frame at any specific point in time. For example,
the endpoint may want to wait for until it is not the only frame to send."